disperser/dataapi/v2/server_v2.go

```diff
@@ -164,10 +163,6 @@ type (
 
 	Metric struct {
 		Throughput float64 `json:"throughput"`
```

Comment:
Are we still using this in dataapi v2?

Reply:
Yes, it's returned

Reply:
How is the throughput being computed for V2?

Reply:
same as v1, prometheus

Reply:
Ok it seems we should update the query it's using since it looks like it's just returning the v1 throughput.

Reply:
Good point.